net/filters.go

```diff
@@ -1,40 +0,0 @@
-package net
```

Comment:
I'm sure we'll want this code again in the future, at which point we can restore it from the git history.